spec/features/guest_user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.feature "GuestUsers", type: :feature do
+  describe("As a guest user") do |variable|
+
```

Comment:
So, I moved this test to the features folder since it was in a feature folder and that didn't seem conventional. Is this correct?